### ISSUE TITLE
Accessibility fixes pt. 1

### ISSUE THREAD

**Issue by gmrabian** [coforma/swift-tech-challenge#167]
**Describe the bug**
We have a number of accessibility concerns we would like to address. This is starting those fixes.

** Current behavior **
1. H1 appears below other page content on apply and confirmation pages
2. Multiple <main> tags on some pages
3. "Apply" and "View more" buttons don't indicate what you are applying for or viewing
4. Details page could use headings better for card navigation—h2s for card titles and h3s for large black text
5. Required application fields should say required for screen readers (instead of just an asterisk)
6. Essay question guidance should appear once so as to not be cumbersome for screen readers
7. Spinner should have text to indicate what is happening
8. Main navigation should have a clear name like "Main navigation" instead of `main-header-nav navigation`

**To Reproduce**
1. View each page as described above
2. Verify with a screen reader or your browser's inspect tools

**Expected behavior**
1. H1 appears as top item on apply and confirmation pages
2. Single <main> tag pair on each page
3. "Apply" and "View more" buttons now state which institution they go with (e.g. "Apply to Penn State University")
4. Details page uses h2s for card titles and h3s for large black text
5. Required application fields have text (required) after the field name
6. Essay question guidance appears once above all essay questions
7. Spinner has text indicating content is loading
8. Main navigation has a clear name: "Main navigation" instead of `main-header-nav navigation`

## Definition of Done
[Definition of Done](https://github.com/coforma/swift-tech-challenge/blob/main/documentation/product/definition_of_done.md)
